src/main/java/lotto/domain/WinningRank.java

```diff
@@ -0,0 +1,44 @@
+package lotto.domain;
+
+import java.util.Arrays;
+
+import static lotto.domain.Utils.makeRangeNumbers;
+
+public enum WinningRank {
+    FAIL(makeRangeNumbers(0, 3), 0),
+    FOURTH(makeRangeNumbers(3, 4), 5000),
```

Comment:
```suggestion
    FOURTH(makeRangeNumbers(3, 4), 5_000),
```
Java 7 부턴 언더스코어르 활용하여 숫자의 가독성을 높일수 있습니다 👍